packages/shared/src/utils/Object.js

```diff
@@ -109,6 +109,10 @@ export const isEqual = (a: any, b: any, userOptions: OptionsProps = {}) => {
   return false;
 };
 
-export default {
```

Comment:
This was the cause of the error being thrown by DataList. We were exporting an empty default object, the re-exporting that in `index.js`. As a convention, we should probably stick to using default exports over named exports. I've updated Browser, which was the only other class I could find using named exports.